rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
The bracket syntax would work in generated ebuilds (e.g automatically generated meta ebuilds or whatnot) if they are generated as EAPI-2 or newer (preferably EAPI-6 right now), but not when passed straight to emerge